src/SolutionTemplate/UnoSolutionTemplate.WinUI.net6/Skia.Linux.FrameBuffer/Program.cs

```diff
@@ -1,5 +1,7 @@
-﻿using System;
+﻿using Microsoft.UI.Xaml;
+using System;
 using Uno.UI.Runtime.Skia;
+using Windows.UI.Core;
```

Comment:
nit: I think most developers keep Visual Studio default of "placing System directives first"

https://github.com/dotnet/roslyn/blob/3e39dd3535962bf9e30bd650e4ff34b610b8349a/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/AddImport/AddImportPlacementOptions.cs#L21-L22

```suggestion
using System;
﻿using Microsoft.UI.Xaml;
using Uno.UI.Runtime.Skia;
using Windows.UI.Core;
```

Reply:
This is the default order from visual studio, as far as I can tell (maybe roslyn behaves differently by default though).